tools/Common.Netcore.Dependencies.targets

```diff
@@ -4,7 +4,7 @@
     <PackageReference Include="Microsoft.Rest.ClientRuntime" Version="2.3.20"/>
     <PackageReference Include="Microsoft.Rest.ClientRuntime.Azure" Version="3.3.19"/>
     <PackageReference Include="Microsoft.Azure.PowerShell.Clients.Aks" Version="1.3.2-preview"/>
-    <PackageReference Include="Microsoft.Azure.PowerShell.Authentication.Abstractions" Version="1.3.2-preview"/>
+    <PackageReference Include="Microsoft.Azure.PowerShell.Authentication.Abstractions" Version="1.3.4-preview"/>
```

Comment:
Cannot upgrade refer to Microsoft.Azure.PowerShell.Authentication.Abstractions only

Reply:
could you please explain more details? is there any other version change needed? Thanks.

Reply:
> @msJinLei could you please explain more details? is there any other version change needed? Thanks.

All package references to common lib of 1.3.2 are expected to be upgraded to 1.3.4. not Microsoft.Azure.PowerShell.Authentication.Abstractions only. We will do the upgrading.

Reply:
This change is no longer needed. Please rebase you patch with master.

Reply:
I update azEnvironmwnt to add attestation endpoint , why you think the pr is not needed?

Reply:
@msJinLei the rebase is finished.